src/views/studio/studio-activity.jsx

```diff
@@ -1,5 +1,6 @@
 import React, {useEffect} from 'react';
 import PropTypes from 'prop-types';
+const FormattedMessage = require('react-intl').FormattedMessage;
```

Comment:
Since the rest of the file uses `import`, this line should be changed to 
```
import {FormattedMessage} from 'react-intl';
```

Reply:
fixed!